src/bindings/python/tests/test_runtime/test_model.py

```diff
@@ -146,22 +146,40 @@ def test_get_result_index_invalid():
     assert model.get_result_index(invalid_output) == -1
 
 
-def test_parameter_index():
-    input_shape = PartialShape([1])
-    param = ops.parameter(input_shape, dtype=np.float32, name="data")
-    relu = ops.relu(param, name="relu")
-    model = Model(relu, [param], "TestModel")
-    assert model.get_parameter_index(param) == 0
-
-
-def test_parameter_index_invalid():
-    shape1 = PartialShape([1])
-    param1 = ops.parameter(shape1, dtype=np.float32, name="data1")
+@pytest.mark.parametrize("shapes, relu_names, model_name, expected_outputs_length, is_invalid, expected_result_index", [
+    ([PartialShape([1])], ["relu"], "TestModel", 1, False, 0),
+    ([PartialShape([1]), PartialShape([4])], ["relu1", "relu2"], "TestModel1", 1, True, -1)
+])
+def test_result_index(shapes, relu_names, model_name, expected_outputs_length, is_invalid, expected_result_index):
+    params = [ops.parameter(shape, dtype=np.float32, name=f"data{i+1}") for i, shape in enumerate(shapes)]
+    relus = [ops.relu(param, name=relu_name) for param, relu_name in zip(params, relu_names)]
+    
+    if is_invalid:
+        model = Model(relus[0], [params[0]], model_name)
```

Comment:
this line may be moved from if-else statement, it's the same

Reply:
done